src/sage/quadratic_forms/genera/genus.py

```diff
@@ -298,13 +298,13 @@ def _blocks(b, even_only=False):
         if not even_only:
             for s in [(1, 2), (5, 6), (1, 6), (5, 2), (7, 0), (3, 4)]:
                 b1 = copy(b)
-                b1[2] = s[0]*(-1)**(rk//2 -1) % 8
+                b1[2] = s[0]*(-1)**(rk // 2 - 1) % 8
                 b1[3] = 1
                 b1[4] = s[1]
                 blocks.append(b1)
             for s in [(1, 4), (5, 0)]:
                 b1 = copy(b)
-                b1[2] = s[0]*(-1)**(rk//2 - 2) % 8
+                b1[2] = s[0]*(-1)**(rk // 2 - 2) % 8
```

Comment:
PEP8 requires spaces around the outermost arithmetic operators only

Reply:
this is one instance of pycodestyle warning E226

Reply:
See also https://github.com/PyCQA/pycodestyle/issues/248